frontend/index.html

```diff
@@ -191,7 +191,8 @@
             <a href="#" onclick="getSpanishPage()">Español</a>
             <script >
               function getSpanishPage(){
-                window.location.href = "index_es.html";
+                const STUDY_NAME_HREF = _getStudyName(window.location);
+                window.location.href = `index_es.html?study_config=${STUDY_NAME_HREF}`;
```

Comment:
hm. this should not be required for the production version, but I agree it is useful for testing.
I anticipate that this can be removed once we get the fully `i18n` version.